### PR TITLE
Update README and CI

### PR DESCRIPTION
**Pull request by abeljoseph**
## Briefly describe the feature or bug that this PR addresses.
Update README.

## Please link the Airtable ticket associated with this PR.
https://airtable.com/tbli2lWQHAqBa6ZcI/viwSkpZmJR8D99kla/recbKtLwhee97VY07?blocks=hide

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -155,3 +155,16 @@ Confirm that the data has indeed been migrated by checking pgAdmin 4.
...
+## Continuous Integration
+The following commands are run with CI:
+```
+pip install -r requirements.txt
+python manage.py test
+```
+The full configuration is found [here](.github/workflows/ci.yml).
+
+## Continuous Deployment
+Deployment is conducted server-side. The documentation can be found [here](https://github.com/serotracker/scripts#update_backendsh).
```
Comment (atatmaja) [inline README.md:170]: Can you include some documentation around how we've made the update_backend script run every 5 mins (i.e. command used to setup the cron job, how to view information about it and configure it for the future, etc)?

Also afaik you have some alerting/monitoring setup to view CI/CD results. If that's true, can you provide some details about that?

**Comment by atatmaja** [comment]
Where could I find info about how the ETL is run on our prod machine?

**Review by atatmaja — APPROVED** [review]
Thanks for writing this up! Please see comments

**Review by atatmaja — CHANGES_REQUESTED** [review]
sorry meant to request changes

**Review by simonarocco — CHANGES_REQUESTED** [review]
can you add documentation please for how one would go about setting up access to the staging and dev machines? everything from the credentials we would need to what commands we would run to ssh into the machine. and then also commands for how to navigate the machines once we are in. e.g. where would we find the logs for the etl and the backend app? where would we find the scripts that deal with automatic deployment and automatically running the etl? maybe don't put some of the sensitive info in the readme but somewhere

**Review by simonarocco — APPROVED** [review]
lgtm, just add the extra things that we discussed on sunday

**Review by atatmaja — APPROVED** [review]

**Pull request merged**